less/build_vizor2teaser.less

```diff
@@ -164,17 +175,26 @@ body {
 .hero__body {
     line-height: 24px;
 }
+// Allow scrolling the page when dragging with finger over text on mobile devices
+@media (max-width: (@desktop-width - 1px)) {
```

Comment:
I think since these all use a flavour of webkit (android and ios), you should also `and (pointer:coarse)` here.